bucket/openclosedriveeject.json

```diff
@@ -0,0 +1,63 @@
+{
+    "version": "1.33",
+    "description": "Eject, open, or close any drive: USB, DVD/CD, SSD, etc.",
+    "homepage": "http://www.softwareok.com/?seite=Microsoft/OpenCloseDriveEject",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://www.softwareok.com/?seite=Microsoft/OpenCloseDriveEject/Eula"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://www.softwareok.com/Download/OpenCloseDriveEject_x64.zip",
+            "hash": "7e48467153d2f0c441c4a55fbe3f47ec8ee4b3629bf45c3096b11b4a0838a7f7",
+            "bin": [
+                [
+                    "OpenCloseDriveEject_x64_p.exe",
+                    "OpenCloseDriveEject"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "OpenCloseDriveEject_x64_p.exe",
+                    "OpenCloseDriveEject"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://www.softwareok.com/Download/OpenCloseDriveEject.zip",
+            "hash": "fa09f13605c944efd606a40b1a267633bf3baa5a885c52fb52de75d3a81d15d8",
+            "bin": [
+                [
+                    "OpenCloseDriveEject_p.exe",
+                    "OpenCloseDriveEject"
+                ]
+            ],
```

Comment:
```suggestion
            "bin": "OpenCloseDriveEject.exe",
```

Reply:
Thanks for catching the x32 difference in filenames!